state/migration_import.go

```diff
@@ -53,6 +53,10 @@ func (st *State) Import(model description.Model) (_ *Model, _ *State, err error)
 		return nil, nil, errors.Trace(err)
 	}
 
+	if len(model.RemoteApplications()) != 0 {
+		return nil, nil, errors.New("can't import models with remote applications")
```

Comment:
could you add a comment here saying why - that CMR is currently limited to models on the one controller